src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -14,7 +14,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Certificates.Configuration.Sources" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
```

Comment:
No longer used?
@javiercn are you going to use Microsoft.AspNetCore.Certificates.Configuration.Sources in Identity? If not, should it be removed from Hosting for now?